src/program.js

```diff
@@ -388,6 +388,13 @@ Example: $0 --help run.
         demand: false,
         type: 'boolean',
       },
+      'experiments': {
+        describe: 'Install one or more WebExtensions Experiments ' +
+                  'alongside the WebExtension Addon.',
```

Comment:
I think you could just say:

    Install one or more WebExtension Experiments alongside the main WebExtension.